tests/engines/registries/acceptance/overview/overview-test.ts

```diff
@@ -433,4 +434,22 @@ module('Registries | Acceptance | overview.overview', hooks => {
         // @ts-ignore
         assert.deepEqual(reg.nodeLicense!.copyright_holders, ['Jane Doe', 'John Doe']);
     });
+
+    test('overview: column layout', async assert => {
+        // 3-column layout
+        const publicReg = server.create('registration', {
+            registrationSchema: server.schema.registrationSchemas.find('prereg_challenge'),
+        });
+
+        await visit(`/${publicReg.id}/`);
+        await percySnapshot(assert);
+
+        // 1-column layout
+        const withdrawnReg = server.create('registration', {
+            registrationSchema: server.schema.registrationSchemas.find('prereg_challenge'),
+        }, 'isWithdrawn');
+
+        await visit(`/${withdrawnReg.id}/`);
+        await percySnapshot(assert);
```

Comment:
This snapshot will not be saved. If you are going to have more than one snapshot in a test, you need to give each a unique name.